src/main/java/ee/buerokratt/ruuter/util/HttpUtils.java

```diff
@@ -23,7 +24,25 @@ public class HttpUtils {
     private HttpUtils() {
     }
 
-    public static HttpResponse<String> makeHttpRequest(HttpQueryArgs args) {
+    public static HttpResponse<String> makeHttpPostRequest(HttpQueryArgs args) {
+        try {
+            ObjectMapper mapper = new ObjectMapper();
+            HttpRequest.Builder request = HttpRequest.newBuilder()
+                .uri(new URI(getUriFromArgs(args)))
+                .timeout(Duration.of(10, SECONDS))
+                .headers("Content-Type", "text/plain;charset=UTF-8")
+                .POST(HttpRequest.BodyPublishers.ofString(mapper.writeValueAsString(args.getBody())));
```

Comment:
We already have a method in mappingUtils that does the same thing - use that one instead of creating a new objectmapper here (HttpUtil shouldn't have to consern itself with mapping objects)